rocky/scheduler.py

```diff
@@ -115,9 +116,28 @@ def __init__(self, base_uri: str):
         self.session = requests.Session()
         self._base_uri = base_uri
 
-    def list_tasks(self, queue_name: str, limit: int) -> PaginatedTasksResponse:
-        params = {"limit": limit}
-        res = self.session.get(f"{self._base_uri}/schedulers/{queue_name}/tasks", params=params)
+    def list_tasks(
+        self,
+        scheduler_id: str,
+        type: Optional[str] = None,
```

Comment:
Be careful: `type shadows built-in name 'type'`

Best to name it differently: e.g. `type_`, `object_type` or `ooi_type`